Update to bevy 0.13

TheNullicorn opened the pull request:
Updates bevy, bevy_egui, and bevy_pancam dependencies for bevy 0.13.

Also fixes the asteroids example, whose shader was broken by bevy 0.13 [rearranging bind groups](https://bevyengine.org/news/bevy-0-13/#wgpu-0-19-upgrade-and-rendering-performance-improvements).

Note: This is a draft because bevy_pancam doesn't yet support bevy 0.13, though [bevy_pancam#56](https://github.com/johanhelsing/bevy_pancam/pull/56) will fix this. For the time being, I've swapped in that PR's github link for the bevy_pancam dependency.

TheNullicorn committed:
Update dependencies

- bevy: 0.12 -> 0.13
  - Move material data from bind group 1 to 2 in asteroid example (as per https://bevyengine.org/news/bevy-0-13/#wgpu-0-19-upgrade-and-rendering-performance-improvements)
- bevy_equi: 0.23 -> 0.25
- bevy_pancam: 0.10 -> 0.11 (waiting on bevy_pancam#56)

Signed-off-by: Nullicorn <git@nullicorn.me>

johanhelsing commented:
Thanks for the fix and upgrade!

I've merged the pancam pr now.

I plan on doing releases on Thursday for most of my bevy crates

johanhelsing commented:
bevy_pancam 0.11 is release now :)

TheNullicorn committed:
Replace `Quad` with `Rectangle` in asteroids example

- Bevy 0.13 deprecated bevy_render::mesh::shape in favor of bevy_math::primitives
  - https://bevyengine.org/news/bevy-0-13/#primitive-shapes

Signed-off-by: Nullicorn <git@nullicorn.me>

TheNullicorn committed:
Update bevy_pancam dependency

- bevy_pancam: 0.10 -> 0.11
  - Now supports bevy 0.13

Signed-off-by: Nullicorn <git@nullicorn.me>

TheNullicorn commented:
Alright, I've updated it in Cargo.toml + caught a deprecation warning I missed earlier.

johanhelsing reviewed (approved):
Excellent, thanks!

sjmueller commented on Cargo.toml:
```diff
@@ -10,19 +10,18 @@ repository = "https://github.com/johanhelsing/noisy_bevy"
 version = "0.5.0"
```
`version = "0.6.0"`

sjmueller commented on Cargo.toml:
```diff
@@ -10,19 +10,18 @@ repository = "https://github.com/johanhelsing/noisy_bevy"
-bevy_pancam = { version = "0.10", features = ["bevy_egui"] }
+bevy_egui = { version = "0.25", default-features = false, features = ["default_fonts"] }
+bevy_pancam = { version = "0.11", features = ["bevy_egui"] }
 insta = "1.21"
```
Can be safely upgrade to `insta = "1.35"`
